### ISSUE TITLE
compute server serial console: rendered *above* scrollbar

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#7359]
for me here in chrome, the serial console output is rendered above the vertical scrollbar. therefore, I can't drag it.

workaround: scrolling with the mouse wheel works as expected.

![screenshot-cocalc com-2024 03 14-10_22_06](https://github.com/sagemathinc/cocalc/assets/207405/fc0d39ad-f290-4283-8d86-e528086febd8)

**Comment by williamstein** [comment]
WORKAROUND: make your screen wider

**Comment by williamstein** [comment]
I can't reproduce this.  Even with scrollbars always on, I don't see this.  I tried many screen widths etc.

<img width="783" alt="image" src="https://github.com/user-attachments/assets/5d993869-da73-421b-995e-a4bdc914c93b">

Maybe it only happened with a different termjs renderer that we stopped using (the opengl) one?  I'm closing this unless you have further info or can confirm it happens, etc.
